test/integration/param_passing.cc

```diff
@@ -61,8 +61,30 @@ TEST(ParamPassingTest, ExperimentalParamsElement)
   errors = root.Load(testFile);
   print_errors(errors);
 
-  // TODO(jenn) update in future
   EXPECT_FALSE(errors.empty());
-  ASSERT_EQ(errors.size(), 1u);
-  EXPECT_EQ(errors[0].Code(), sdf::ErrorCode::ELEMENT_INVALID);
+  ASSERT_EQ(errors.size(), 2u);
+  EXPECT_EQ(errors[0].Code(), sdf::ErrorCode::ATTRIBUTE_MISSING);
+  EXPECT_EQ(errors[1].Code(), sdf::ErrorCode::ELEMENT_MISSING);
+}
+
+/////////////////////////////////////////////////
+TEST(ParamPassingTest, NestedInclude)
+{
+  const std::string modelRootPath =
+    sdf::filesystem::append(PROJECT_SOURCE_PATH, "test", "integration",
+                            "model", "nested_include") + "/";
+  sdf::setFindCallback(
+      [&](const std::string &_file)
+      {
+        return modelRootPath + _file;
+      });
+
+  std::string testFile =
+    sdf::filesystem::append(PROJECT_SOURCE_PATH, "test", "integration", "model",
+                            "nested_include", "test_nested_include.sdf");
+  sdf::Root root;
+  sdf::Errors errors = root.Load(testFile);
+  print_errors(errors);
+
+  EXPECT_TRUE(errors.empty());
```

Comment:
Maybe add a test for error cases, such as missing a `name` attribute in an `experimental:params` child element?

Reply:
There is this [failure case](https://github.com/osrf/sdformat/blob/eba07eb93136c460d0c6e43e0201da4b7af0c23c/test/integration/param_passing.cc#L66) for missing a `name` in an `experimental:params` child element using this [sdf file](https://github.com/osrf/sdformat/blob/eba07eb93136c460d0c6e43e0201da4b7af0c23c/test/integration/include_invalid_custom_model.sdf#L8). Is that what you meant or did you mean something else?

Reply:
That's good, thanks.